src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -2322,8 +2348,7 @@ subroutine internal_tides_init(Time, G, GV, US, param_file, diag, CS)
   use_temperature = .true.
   call read_param(param_file, "ENABLE_THERMODYNAMICS", use_temperature)
   if (.not.use_temperature) call MOM_error(FATAL, &
-    "register_int_tide_restarts: internal_tides only works with "//&
```

Comment:
Not part of this PR, but does `ENABLE_THERMODYNAMICS` need a `do_not_log`?

Reply:
There are other instances of this in the code, and may not even be a problem.  If it is, then we can address this later.